### PR TITLE
Show drinks

### LINKED ISSUE #2: Show Drinks
Two buttons should be over the pizza list. You can switch between drink and pizza list with these buttons.

Here is the end point for drinks:https://doclerlabs.github.io/mobile-native-challenge/drinks.json

A list element should contain the name of the drink and the price and a button which we can add to cart.

### PR DESCRIPTION
**Pull request by InstantBuddha**
Added:

1. a top menu to select pizzas, drinks and the cart
2. working list of drinks

### CODE REVIEW

File: src/components/DrinksLister.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            this.setState({drinks: response.data,
+                            ready: true})
+        })
+    }
+    
+    render() {
+        const drinks = this.state.drinks
+        const bootstrapCss = "d-flex flex-column align-items-center"
+
+        console.log(this.state)
```
Comment (gavrillas) [inline src/components/DrinksLister.js]: Remove the console log

File: src/components/DrinksLister.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        return (
+            <div className={bootstrapCss}>
+                {
+                    this.state.ready ?
+                    drinks.map(
+                        drink => <DrinkCard 
+                            key={drink.name}
```
Comment (gavrillas) [inline src/components/DrinksLister.js]: I would rather use drink.id as a key.

File: src/components/MainWrapper.js [hunk 3]
```diff
@@ -11,22 +14,43 @@ class MainWrapper extends Component {
...
+        copiedTempState.whatToShow = toDisplay
+        this.setState(copiedTempState)
+    }
 
+    listerReturner(){
+        switch(this.state.whatToShow){
+            case "pizzas":
+                return <PizzasLister productCartAdder={this.productCartAdder} />
+            case "drinks":
+                return <DrinksLister />
```
Comment (gavrillas) [inline src/components/MainWrapper.js]: You could add productCartAdder to DrinksLister as well. Stay consistent.

File: src/components/DrinksLister.js (hunk 1 above)
Comment (InstantBuddha) [reply src/components/DrinksLister.js]: Done :)

File: src/components/DrinksLister.js (hunk 2 above)
Comment (InstantBuddha) [reply src/components/DrinksLister.js]: Done :)

**Pull request merged**